chalice/app.py

```diff
@@ -17,10 +17,25 @@
 import requests
 from flask import json
 
-
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'chalicelib'))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
+XRAY_TRACE = os.environ.get('XRAY_TRACE', 'false') == 'true'  # noqa
+
+if XRAY_TRACE:  # noqa
```

Comment:
If you want to instrument all of the lambdas you'll need to add this to all of the `app.py`s in `daemons/` right after `Domovoi` is called. Maybe we should add x-ray tracing hook to `Domovoi` or something to avoid lots of duplicate code.